src/main/java/com/example/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,29 @@
 package com.example.bookstore.dto;
 
+import com.example.bookstore.validator.Isbn;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Pattern;
+import jakarta.validation.constraints.PositiveOrZero;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.Length;
 
 @Data
 public class CreateBookRequestDto {
+    @NotBlank(message = "Can't be null or empty")
+    @Length(min = 3, max = 50, message = "Length must be more than  3, but less than  50")
+    @Pattern(regexp = "^[a-zA-Z0-9\\s]+$", message = "Only Latin letters and digits are allowed")
```

Comment:
What about special symbols? Might not title contain full stops, colons, asterisks, or others?